README.md

```diff
@@ -7,67 +7,98 @@ Perses
 
 ## Overview
 
-Perses is part of the [CoreDash community](https://github.com/coredashio/community).
+Perses is part of the [CoreDash community](https://github.com/coredashio/community). It belongs to the Linux Foundation.
+On a later stage, we want to promote the project to the [Cloud Native Computing Foundation](https://www.cncf.io/) and be
```

Comment:
"At a later stage" (I would suggest edits but when those get accepted it breaks the DCO)